grammar.js

```diff
@@ -625,7 +642,7 @@ module.exports = grammar({
       optional(
         $.keyword_only,
       ),
-      $.table_reference,
+      $.object_reference,
```

Comment:
We should not forget to fix the queries downstream in nvim-treesitter. There is a query matching on `table_reference`